test/Setup.t.sol

```diff
@@ -650,8 +821,22 @@ contract Setup is Test {
         paramValue = uint248(uint256(extraParams_));
     }
 
-    // to ignore this file from coverage
+    // to test contract setup on one chain
+    ChainContext aTestChain;
+
     function test() external {
-        assertTrue(true);
+        initialise();
+        uint256[] memory transmitterPivateKeys = new uint256[](1);
+        transmitterPivateKeys[0] = _transmitterPrivateKey;
+
+        aTestChain.chainSlug = uint32(10);
+        uint32 bTestChainSlug = uint32(20);
+
+        _deployContractsOnSingleChain(
+            aTestChain,
+            bTestChainSlug,
+            isExecutionOpen,
+            transmitterPivateKeys
+        );
```

Comment:
This test runs for each test file. Can we change somehow to only run once?

Reply:
removing it, not testing anything rn